Lib9c/Model/State/RankingState.cs

```diff
@@ -53,7 +53,22 @@ public Address UpdateRankingMap(Address avatarAddress)
             throw new RankingExceededException();
         }
 
-        public override IValue Serialize() =>
+        public override IValue Serialize()
+        {
+#pragma warning disable LAA1002
+            var rankingMapValue = new Bencodex.Types.Dictionary(RankingMap.Select(pair =>
+#pragma warning restore LAA1002
+                new KeyValuePair<IKey, IValue>(
+                    (Bencodex.Types.Binary)pair.Key.Serialize(),
+                    new Bencodex.Types.List(pair.Value
+                        .OrderBy(e => e.GetHashCode())
+                        .Select(e => e.Serialize())))));
+            return ((Bencodex.Types.Dictionary)base.Serialize())
+                .SetItem("ranking_map", rankingMapValue);
+        }
+
+        [Obsolete("Since it contains a deterministic problem, replace it with the `Serialize()` method that operates deterministically like this logic.")]
```

Comment:
I guess if there isn't any usage for `SerializeV1()` and new `Serialize()` acts as previous, we don't need to preserve the previous code.

Reply:
@longfin I applied this.